cmd/sys-mgmt-agent/Dockerfile

```diff
@@ -1,48 +1,58 @@
 #
 # Copyright (c) 2018
-# Dell
 # Cavium
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
 FROM golang:1.11.2-alpine3.7 AS builder
```

Comment:
The go.sum will trip up the build if the golang version varies.  Here we are pegged at 1.11.2,
You should spec the base image like this `FROM golang:1.11-alpine AS builder` to conform to the other edgex-go services (this pulls in the latest right now - 1.11.5)
@tsconn23

Reply:
Gotcha @JPWKU - I _indeed_ have `FROM golang:1.11.2-alpine3.7 AS builder`, which I'll modify to conform to `FROM golang:1.11-alpine3.7 AS builder` 👍